src/libImaging/Pack.c

```diff
@@ -648,8 +648,10 @@ static struct {
     {IMAGING_MODE_I_16B, IMAGING_RAWMODE_I_16B, 16, copy2},
     {IMAGING_MODE_I_16L, IMAGING_RAWMODE_I_16L, 16, copy2},
     {IMAGING_MODE_I_16N, IMAGING_RAWMODE_I_16N, 16, copy2},
-    {IMAGING_MODE_I_16, IMAGING_RAWMODE_I_16N, 16, packI16N_I16
-    },  // LibTiff native->image endian.
+    {IMAGING_MODE_I_16,
```

Comment:
This is actively bad formatting, destroying all of the symmetry of the lines above and below.

Reply:
I've added more changes to rearrange the code.